pkg/utils/helper.go

```diff
@@ -59,20 +59,30 @@ func RunCommandOnHost(command string, arg ...string) (string, error) {
 	return string(out), nil
 }
 
-// RunCommandOnContainer runs a command on container system
-func RunCommandOnContainer(command string, arg ...string) (string, error) {
+// RunCommandOnContainerBase runs a command on container system with given stdout and stderr buffers

```

Comment:
I think this should be a question for the periscope team - they may have objections to this since it's used by other collectors as well